### PR TITLE
Implements ZipFinder

### PR DESCRIPTION
**Pull request by ludanin**
These sets of commits fixes a small issue at Blurb, changes how the viewport height is calculated, and implements small feature to add to the App: the ability to let users find their Zip-Code without having to leave the page. 

On Blurb a small typo in the styled CSS is corrected (see 14e1a20) and the logic to get 100vh in mobile devices is converted to a CSS Variable and initialized at client/src/index.tsx, in order to allow all components to use this variable without the need of Hooks/States (see 
8285493).

ZipFinder is a new component that lets users search for their Zip-Code within the app. The component follows the Material Design Guidelines and hides itself when the user doesn’t need it.

For these changes to take place two new packages (`react-geosuggest` and `react-toastify`) were added in the client side of the project. 

`react-geosuggest` allows us to fetch Zip-Codes through the aid of Google Maps API, and `react-toastify` provides a ready-to-use notification system that is very easy to implement on an existing application.

Remember to insert a `script` tag to the `public/index.html` located at the `client` folder to allow `react-geosuggest` to fetch information using your Google Maps API Key

e.g.

```html
<script src="https://maps.googleapis.com/maps/api/js?libraries=places&key=GOOGLE_MAPS_API_KEY"></script>
```

### CODE REVIEW

File: packages/client/src/comp/StateContainer.tsx [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
-    </AddressContainer.Provider>
-  </QueryContainer.Provider>
+  <ZipFinderContainer.Provider>
+    <QueryContainer.Provider>
+      <AddressContainer.Provider>
+        <ContactContainer.Provider>
+          <AnalyticsContainer.Provider>
+            <VoterContainer.Provider>
+              {children}
+              <ToastContainer
```
Comment (tianhuil) [inline packages/client/src/comp/StateContainer.tsx:22]: Does `ToastContainer` have to wrap `{children}`?

Comment (ludanin) [reply packages/client/src/comp/StateContainer.tsx:22]: Actually it is supposed to be in this way (when I used this library for the first time that bit surprised me as well).

ToastContainer doesn't render its children, thus we'll always end up with a blank page when passing children inside this Container.

File: packages/client/package.json [hunk 2]
```diff
@@ -9,8 +9,10 @@
     "react-dom": "^16.12.0",
     "react-facebook-pixel": "^0.1.3",
     "react-ga": "^2.7.0",
+    "react-geosuggest": "^2.13.0",
```
Comment (tianhuil) [inline packages/client/package.json:12]: How many KB's does this package contribute?

Comment (ludanin) [reply packages/client/package.json:12]: According to react-geosuggest dist folder, the minified version is about 17kb.
I've also checked the package on [bundlephobia](https://bundlephobia.com/result?p=react-geosuggest@2.13.0) and it seems that tree shaking support is availbale when using webpack.

**Pull request closed without merging**